### PR TITLE
Travis ci  and coveralls config

### PR DESCRIPTION
**Pull request by gilgamezh**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,14 +39,18 @@
         'sgp4',
         'requests',
     ],
-    tests_require=['logassert', 'flake8', 'hypothesis', 'mock', 'hypothesis[datetime]'],
```
Comment (Juanlu001) [inline setup.py]: Do we need to add the requirements in both places? AFAIK `tests_require` is only used by setuptools when doing `python setup.py test`, and downloads the .eggs to a hardcoded location using `easy_install`. It's a bit painful (I'm thinking of removing it from all of my projects), so if you think we can remove them, I fully support doing so :)

File: README.rst [hunk 2]
```diff
@@ -1,10 +1,16 @@
 Orbit Predictor
 ===============
 
+.. image:: https://travis-ci.org/satellogic/orbit-predictor.svg?branch=master
+    :target: https://travis-ci.org/satellogic/orbit-predictor
+.. image:: https://coveralls.io/repos/github/satellogic/orbit-predictor/badge.svg?branch=master
+:target: https://coveralls.io/github/satellogic/orbit-predictor?branch=master
```
Comment (Juanlu001) [inline README.rst]: Indentation?

File: setup.py (hunk 1 above)
Comment (gilgamezh) [reply setup.py]: +1 thanks!

Comment (gilgamezh) [reply setup.py]: I changed it, I need you to approve the MR to merge it

Comment (Juanlu001) [reply setup.py]: For my review to unblock the merge I need write access to the repo!

**Comment by Juanlu001** [comment]
This can now be merged :)

**Pull request merged**